test/data/readywait.html

```diff
@@ -14,21 +14,29 @@
 		#output { background-color: green }
 		#expectedOutput { background-color: green }
 	</style>
-	<script src="jquery.js"></script>
+	<script src="../jquery.js"></script>
+	<script src="./iframeTest.js"></script>
 
 	<!-- Load the script loader that uses
 		jQuery.readyWait -->
-	<script src="data/readywaitloader.js"></script>
 
 	<script type="text/javascript">
+	// Hold on jQuery!
+	jQuery.holdReady( true );
+
+	setTimeout( function() {
+		jQuery.holdReady( false );
+	}, 2000 );
```

Comment:
Two seconds is a lot of idle time to add for this test. If you take my advice below about assigning to a variable here, we can instead set a short timeout and just push it back if it fires before `jQuery.ready` has been invoked.